src/sql_output/utils.ts

```diff
@@ -1,23 +1,26 @@
-import * as vscode from "vscode";
+import {
+  ExtensionContext,
+  Uri
+} from 'vscode';
 
 export interface CompilationResult {
-  status: "ok" | "error";
+  status: 'ok' | 'error';
```

Comment:
If we should add a formatter for these, lmk, we have loads of devops in the main repo, but I haven't spent that much time with this repo

Reply:
no big deal. I wanted to cleanup vscode imports and decided to do strings too.